### DISCUSSION TITLE
Where are the FSharp.Core 7.0.300 release notes?

### DISCUSSION THREAD

**Discussion by cataggar** [dotnet/fsharp#15332] [Q&A] [answered]
I'd like to know what it is in the 7.0.300 and why I should update to it in our applications.

https://www.nuget.org/packages/FSharp.Core/7.0.300#release-body-tab
Provides a link to:
https://dnceng@dev.azure.com/dnceng/internal/_git/dotnet-fsharp/blob/main/release-notes.md#FSharp-Core-7-0-300

![image](https://github.com/dotnet/fsharp/assets/87583576/bb95a21b-fdf4-45f6-9cf4-b5620bdbbb8f)

I also do not see `7.0.300` listed here:
https://github.com/dotnet/fsharp/blob/main/release-notes.md

**Comment by vzarytovskii** [comment, marked as answer]
We don't generate release notes for `FSharp.Core` at this moment, mostly due to external release process.
Version bump also doesn't mean there are any changes (we bump version every time we branch from previous release), your best choice now will probably be to look at public API surface changes:
https://www.fuget.org/packages/FSharp.Core/7.0.300/lib/netstandard2.1/diff/7.0.200/

Other changes are probably internal implementation details.

**Comment by cataggar** [comment]
Thanks. Not much has changed externally. https://www.fuget.org/packages/FSharp.Core/7.0.300/lib/netstandard2.1/diff/6.0.7/

**Reply by vzarytovskii** [reply]
Yeah, many changes are internal details and optimizations. We'll try to figure out a way of how to publish those, but might take some time.
